src/index.ts

```diff
@@ -66,6 +67,9 @@ function parseXPath(xpathString: string) {
 	}
 
 	const ast = parseExpression(xpathString, { allowXQuery: false });
+
+	annotateAst(ast);
```

Comment:
index should not be changed

Reply:
Were just adding annotation here so that should be fine

Reply:
The imports should not be changed, our content can be changed.